app/src/main/java/com/github/se/polyfit/ui/screen/OverviewScreen.kt

```diff
@@ -72,6 +73,12 @@ fun OverviewScreen(
   val iconExample = BitmapFactory.decodeResource(context.resources, R.drawable.picture_example)
   var imageBitmap by remember { mutableStateOf(iconExample) }
 
+  var mealSummary by remember { mutableStateOf(listOf<Pair<MealOccasion, Double>>()) }
```

Comment:
I think It would be good to create a LiveData variable in the viewModel to observe as state in the overviewScreen

Reply:
That is a great idea and is already happening. On line 78:

```kotlin
  LaunchedEffect(Unit) {
    overviewViewModel.getCaloriesPerMealOccasionTodayLiveData().observeForever { mealSummary = it }
  }
```

An observer is set the first time the screen is displayed on the LiveData. The LiveData is not created at the level of the overviewViewModel but at the level of the database.